### DISCUSSION TITLE
IS BEVE portable for other platform？like ARM

### DISCUSSION THREAD

**Discussion by wanghenshui** [beve-org/beve#12] [Q&A] [answered]
I saw README

> The endianness must be little endian

It's mean only support little endian platform OR store as little endian?

**Comment by stephenberry** [comment]


**Reply by wanghenshui** [reply]
just curious. old powerpcs are big endian.

**Reply by stephenberry** [reply]
Yes, though even powerpc eventually started to switch to little endian (offering both and Linux systems often choosing little endian) and I haven't yet run into anyone using a big endian machine. Albeit I'm sure they exist out there, but would probably be using old compilers.

**Comment by stephenberry** [comment, marked as answer]
BEVE can be used on big endian machines. It simply needs to be encoded in little endian format. The Glaze library supports BEVE on big endian machines.
